Core.lua

```diff
@@ -589,9 +589,16 @@ do
 	-- @name IsRowVisible
 	-- @description Checks if a row is currently being shown
 	-- @usage st:IsRowVisible(realrow)
-	-- @thanks sapu94
 	local function IsRowVisible(self, realrow)
-		return (realrow > self.offset and realrow <= (self.displayRows + self.offset))
+		-- Scan through all on-screen rows, checking their real data row numbers for a match.
+		local firstVisibleIdx = 1 + (self.offset or 0);
```

Comment:
This line, explained: Blizzard's FauxScrollingTable's offset begins counting from `0` (meaning "nothing is scrolled out of view"). Whereas array indices (aka the first row) begin counting from `1`. Hence we need: `1 + (self.offset or 0);`. The "or 0" is just a failsafe since in some cases (I think it was before the user has done any scrolling, since lib-st sets its .offset value during the "on vertical scroll" event), Lib-ScrollingTable doesn't update the offset right away when the table is created. But at that time, the offset is zero, so we fall back to zero.